templates/zerver/help/upload-and-share-files.md

```diff
@@ -2,28 +2,45 @@
 
 Zulip allows you to easily attach files, including images, sound, and video by:
 
-- *Clicking the Paperclip Icon* Use the paperclip icon (![paperclip](/static/images/help/paperclip.png)) located under the message field.
-- *Dragging* You may also drag and drop files on the message field. The files are uploaded to a server and a link is supplied, but a thumbnail is displayed if possible. The link to the file will be in Markdown format: `[link_text](link_URL)`
+- *Clicking the Paperclip Icon* Use the paperclip icon
+(![paperclip](/static/images/help/paperclip.png)) located under the message field.
+- *Dragging* You may also drag and drop files into the message field.
+The files are uploaded to a server and a link is supplied,
+but a thumbnail is displayed if possible.
+The link to the file will be in Markdown format: `[link_text](link_URL)`
 
 ![before](/static/images/help/file_upload_before.png)
 
-If you wish to give your image a caption or title, you may add text before or after the generated link. Any text you add in the box will not break the link to your file so long as the file link remains intact.
+If you wish to give your image a caption or title,
+you may add text before or after the generated link.
+Any text you add in the box will not break the link to your file so long as the file link remains intact.
 
 ---
 
-After submitting your message, you will see a clickable thumbnail if your file is a common image type (i.e. PNG, JPEG, etc.). All other files will simply have a link displayed.
+After submitting your message, you will see a clickable thumbnail if your
+file is a common image type (i.e. PNG, JPEG, etc.). All other files will simply
+have a link displayed.
 
 ![after](/static/images/help/file_upload_after_image_preview.png)
 
 ### Link Behavior
 
-Clicking all file links will open a new tab that will either 1. display the file if it is an image or 2. download the file automatically to your computer.
+Clicking all file links will open a new tab that will either
+- display the file if it is an image
+- download the file automatically to your computer.
 
-If the file has a thumbnail, clicking it will open a lightbox modal with varying options depending on the media type. Images may be opened in new tabs or downloaded, while videos from Youtube may only be opened (and not downloaded). You can exit out of a lightbox by clicking the space around the media preview or by clicking the exit button found in the upper right-hand corner.
+If the file has a thumbnail,
+clicking it will open a lightbox modal with varying options
+depending on the media type. Images may be opened in new tabs or downloaded,
+while videos from Youtube may only be opened (and not downloaded).
+You can exit out of a lightbox by clicking the space around the media
+preview or by clicking the exit button found in the upper right-hand corner.
```

Comment:
You can add an image icon for the exit button `((![x](/static/images/help/x.png))`